docs/deployment-customizations.md

```diff
@@ -54,9 +54,9 @@ ec2_os_distro = "<rhel>"
 ec2_ami_id    = "<custom-rhel-ami-id>"
```

Comment:
## Custom AMI
By default, this module will use the standard AWS Marketplace image based on the value of the `ec2_os_distro` input (either `ubuntu`, `rhel`, or `al2023`). 

```hcl
ec2_os_distro = "<rhel>"
```
If you prefer to use your own custom AMI, then you may do so by setting `ec2_ami_id` accordingly.
If you have a custom AWS AMI you would like to use, you can specify it via the following module input variables:
```hcl
ec2_ami_id    = "<custom-ami-id>"
```